react/src/containers/AlbumFormContainer.js

```diff
@@ -27,45 +26,53 @@ handleClearForm(event) {
 
 handleSubmitForm(event){
 event.preventDefault();
-let submissions = [];
-let payload = [this.state.title, this.state.date_released, this.state.artist]
-this.setState({ form: submissions.concat(payload) });
-this.handleClearForm(event)
+let payload = { album: { title: this.state.title, date_released: this.state.date_released, artist_name: this.state.artist } };
+// debugger;
+console.log(payload);
```

Comment:
Likewise this won't break anything but should technically be removed when pushing final code :)